src/main/java/mops/businesslogic/Group.java

```diff
@@ -1,23 +1,21 @@
 package mops.businesslogic;
 
+import lombok.Value;
+
 /**
  * Represents a group (course, study group, etc.).
  */
-public interface Group {
-
-    /**
-     * @return database id
-     */
-    Long getId();
+@Value
+public class Group {
```

Comment:
You need to add `@SuppressWarnings` here. Copy it from the other `@Value` classes ;)